### PR TITLE
Update Anope.

### PR DESCRIPTION
**Pull request by SadieCat**
I also switched it to building with Ninja as that builds a lot faster.

### CODE REVIEW

**Comment by SadieCat** [comment]
Ah it seems like I need to update the Anope controller to use the new linking protocol names. I'll look into that now.

File: irctest/controllers/anope_services.py [hunk 1]
```diff
@@ -101,15 +101,15 @@ def run(self, protocol: str, server_hostname: str, server_port: int) -> None:
...
+        services_path = shutil.which("anope")
         assert services_path
 
         # Config and code need to be in the same directory, *obviously*
         (self.directory / "lib").symlink_to(Path(services_path).parent.parent / "lib")
 
         self.proc = subprocess.Popen(
             [
-                "services",
+                "anope",
```
Comment (progval) [inline irctest/controllers/anope_services.py:112]: nice

File: workflows.yml [hunk 2]
```diff
@@ -343,16 +343,16 @@ software:
...
         path: anope
         refs:
-            stable: "2.0.9"
-            release: "2.0.9"
-            devel: "2.0.9"
-            devel_release: "2.0.9"
+            stable: "2.0.14"
+            release: "2.1.1"
+            devel: "2.1"
+            devel_release: "2.0"
```
Comment (progval) [inline workflows.yml:349]: nice too

**Comment by progval** [comment]
> Ah it seems like I need to update the Anope controller to use the new linking protocol names. I'll look into that now.

It seemed to pass the CI this way, do you mean it is going to be an issue for other versions?

**Pull request merged**